worker/src/DepLibUring.cpp

```diff
@@ -310,7 +313,19 @@ DepLibUring::LibUring::LibUring()
 
 	if (err < 0)
 	{
-		MS_THROW_ERROR("io_uring_register_buffers() failed: %s", std::strerror(-err));
+		if (err == ENOMEM)
+		{
+			this->zeroCopyEnabled = false;
+
+			MS_WARN_TAG(
+			  info,
+			  "io_uring_register_buffers() failed due to low memlock limit (ulimit -l), disabling zero copy",
+			  std::strerror(-err));
```

Comment:
This is not gonna log anything in CI since `logTag` "info" is not enabled and `logLevel` is not set and `DEBUG` env variable is not set.

Reply:
No need to log in CI IMO.